src/fetchtastic/download/android.py

```
@@ -0,0 +1,672 @@
+    """
+    Downloader for Meshtastic Android APK files.
+
+    This class handles:
+    - Fetching Android APK releases from GitHub
+    - Downloading APK files
+    - Managing Android-specific version tracking
+    - Handling Android prereleases
+    - Cleaning up old Android versions
+    """
+
+    def __init__(self, config: Dict[str, Any], cache_manager: "CacheManager"):
```

Ruff (0.14.8)

Undefined name `CacheManager`

(F821)